CHANGELOG.md

```diff
@@ -2,6 +2,96 @@
 
 This document outlines major changes between releases.
 
+## 0.107.0 "Multithreading/Multiplication" (02 Dec 2024)
+
+A large update that except for numerous enhancements and bug fixes introduces a
```

Comment:
`that except` doesn't read well

Reply:
Removed, we have enough details for other features of this release.